scripts/fetch_manifests.sh

```diff
@@ -21,7 +21,7 @@ manifests=(
   replicaset
 )
 
-if [[ "${CAPM3_VERSION}" == "v1alpha4" ]]; then
+if [[ "${CAPM3_VERSION}" == "v1alpha5" ]]; then
```

Comment:
do we need the condition for v1alpha5? if yes , need a `else` for v1alpha4.

Reply:
Do we need condition at all here? Ain't this same for both APIs?